lib/options.js

```diff
@@ -108,6 +108,12 @@ module.exports = optionator({
         description: "Report errors only"
     },
     {
+        option: "max-warnings",
+        type: "Number",
+        default: "-1",
+        description: "Show an error if too many warnings"
```

Comment:
How about something like, "Number of warnings to trigger nonzero exit code."?

Doing forget to update the docs as well.